### PR TITLE
PP-3671 Content changes for on demand mandates

### PR DESCRIPTION
**Pull request by DanailMinchev**
## WHAT

- Make sure that we do not display "payment" when we actually setting up a "madnate"

with @soniaturcotte

### CODE REVIEW

File: app/cancel/get.njk [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {% extends "common/templates/layout.njk" %}
 
 {% block page_title %}
-  Your payment has been cancelled - GOV.UK Pay
+  Your {{ "mandate" if mandateType == 'ON_DEMAND' else "payment" }} has been cancelled - GOV.UK Pay
```
Comment (alexbishop1) [inline app/cancel/get.njk]: I think we’d future-proof ourselves if this was `
"payment" if mandateType == 'ONE_OFF' else "mandate"` — it being a payment rather than a mandate is the exceptional case (and will ensure this code keeps working when we have `RECURRING` mandates or whatever)

File: common/response.js [hunk 2]
```diff
@@ -14,14 +14,8 @@ function renderErrorView (req, res, msg = ERROR_MESSAGE, status = 500) {
   res.status(status)
   res.render('common/templates/error', {'message': msg})
 }
-function renderPaymentCompletedSummary (req, res, params) {
```
Comment (alexbishop1) [inline common/response.js]: Why was this removed?

Comment (DanailMinchev) [reply common/response.js]: It is not used in the project

File: app/cancel/get.njk (hunk 1 above)
Comment (DanailMinchev) [reply app/cancel/get.njk]: As discussed:
In case of `ONE_OFF` we want to display `payment` even when we cancel the one off mandate, because users don't have idea of mandate in one off payments.
That is why the condition `"payment" if mandateType == 'ONE_OFF' else "mandate"` should include all scenarios

File: common/response.js (hunk 2 above)
Comment (alexbishop1) [reply common/response.js]: Fair enough. Ideally this would have in a separate PR but I know how it is.
